doc/kernel_tuning.md

```diff
@@ -0,0 +1,138 @@
+|CONFIG_USB_EMI62 |	EMI 6|2m USB Audio interface support | This driver loads firmware to Emagic EMI 6|2m low latency USB Audio and Midi interface.
+|CONFIG_USB_STORAGE_KARMA |	Support for Rio Karma music player | additional code to support the Rio Karma USB interface.
+|CONFIG_USB_EMI26 |	EMI 2|6 USB Audio interface support | This driver loads firmware to Emagic EMI 2|6 low latency USB Audio interface.
+|CONFIG_USB_ADUTUX |	ADU devices from Ontrak Control Systems | if you want to use an ADU device from Ontrak Control Systems.
+|CONFIG_USB_SEVSEG |	USB 7-Segment LED Display | Say Y here if you have a USB 7-Segment Display by Delcom
+|CONFIG_USB_RIO500 |	USB Diamond Rio500 support | Say Y here if you want to connect a USB Rio500 mp3 player to your computer's USB port.
+|CONFIG_USB_LEGOTOWER |	USB Lego Infrared Tower support | Say Y here if you want to connect a USB Lego Infrared Tower to your computer's USB port.  
+|CONFIG_USB_LCD |	USB LCD driver support | Say Y here if you want to connect an USBLCD to your computer's USB port. The USBLCD is a small USB interface board for alphanumeric LCD modules.
+|CONFIG_USB_CYPRESS_CY7C63 |	Cypress CY7C63xxx USB driver support | Say Y here if you want to connect a Cypress CY7C63xxx micro controller to your computer's USB port.
+|CONFIG_USB_CYTHERM |	Cypress USB thermometer driver support | Say Y here if you want to connect a Cypress USB thermometer device to your computer's USB port. This device is also known as the Cypress USB Starter kit or demo board.
+|CONFIG_USB_IDMOUSE |	Siemens ID USB Mouse Fingerprint sensor support | Say Y here if you want to use the fingerprint sensor on the Siemens ID Mouse.
+|CONFIG_USB_APPLEDISPLAY |	Apple Cinema Display support | Say Y here if you want to control the backlight of Apple Cinema Displays over USB.
+|CONFIG_USB_SISUSBVGA |	USB 2.0 SVGA dongle support (Net2280/SiS315) | Say Y here if you intend to attach a USB2VGA dongle based on a Net2280 and a SiS315 chip.
+|CONFIG_USB_TRANCEVIBRATOR |	PlayStation 2 Trance Vibrator driver support  | Say Y here if you want to connect a PlayStation 2 Trance Vibrator device to your computer's USB port.
+|CONFIG_USB_IOWARRIOR |	IO Warrior driver support | Say Y here if you want to support the IO Warrior devices from Code Mercenaries.
+|CONFIG_USB_EHSET_TEST_FIXTURE |	USB EHSET Test Fixture driver  | Say Y here if you want to support the special test fixture device used for the USB-IF Embedded Host High-Speed Electrical Test procedure.
+|CONFIG_USB_ISIGHTFW |	iSight firmware loading support | This driver loads firmware for USB Apple iSight cameras
+|CONFIG_USB_YUREX |	USB YUREX driver support | Say Y here if you want to connect a YUREX to your computer's USB port. The YUREX is a leg-shakes sensor.
+|CONFIG_USB_ATM |	USB DSL modem support | Say Y here if you want to connect a USB Digital Subscriber Line (DSL) modem to your computer's USB port
```

Comment:
Maybe apply this configuration change in Linux’ config, and then run `make savedefconfig`, and rename `defconfig` to `defconfig.optimized`.